EIPS/eip-3290.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 3290
+title: Time Based Switch
+author: Branislav Djalic (@Omodaka9375), Malisa Pusonja (@horohronos), Ivor Jugo (@ivorrr)
+discussions-to: https://github.com/ethereum/EIPs/issues/3286
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2021-02-23
+---
+
+## Simple Summary
+This EIP provides standard interfaces and processes for safe, conditional transactions based on user predefined timestamps. It would enable users to secure their accounts from lockout after account activity ceased for a predefining fallback address to transfer the funds.
+
+## Abstract
+This EIP standard addresses the user's need to enable the transfer of his fund from an account that he can not access anymore, whatever the reason is. 
+
+We are aware of the complexity and technical knowledge that a user must have to operate with his crypto accounts securely, when we add that the users' costs of time and liability only on themselves for storing their funds, it could be one of the reasons why mass adoption still didn't come.
+
+By providing a trusted decentralized way to trigger this kind of scheduled transaction in the future, this EIP will help to avoid diversification of similar products and establish a universal solution to all future implementations. 
+
+The creator/user is required to fulfill a set of predefined rules when creating a time switch contract (Will go through more details in the specification section):
+
+* Time represents the timestamp between the current moment and the scheduled one. 
+* Assets and amount of the token that he is willing to transfer.
+* Recipients address and the percentage of funds that each one of them will get.
+* Executor address, which can trigger the transactions.
```

Comment:
Abstract should be a human readable summary of the technical changes.  Most of this content looks like stuff that would be more appropriate for the `Motivation` section as it describes *why* someone would want to implement this EIP.  Something like "Defines an interface for functions X and Y along with event Z".

Should be around a paragraph or two ideally, and be written in plain English (though referring to technical terms and functions is acceptable).  Don't go too detailed, and you don't have to describe the entire system, just the high level and most important bits.